packages/examples/cvat/exchange-oracle/src/handlers/job_creation.py

```diff
@@ -1388,23 +1556,20 @@ def build(self):
         # Data preparation
         self._extract_and_upload_rois()
         self._upload_task_meta()
+        self._prepare_gt_roi_dataset()
```

Comment:
It's better to move this before the ROI extraction and uploading call to fail faster in case of errors, if possible.